src/airflow_lite/api/static/css/layout.css

```
@@ -1,186 +1,157 @@
-  margin: 0;
-  padding: 6px 6px 2px;
-  color: var(--sidebar-muted);
-  font-size: 0.6rem;
-  font-weight: 700;
-  letter-spacing: 0.12em;
-  text-transform: uppercase;
-  text-align: center;
+.sidebar-toggle-line {
+  width: 12px;
+  height: 1.5px;
+  background: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)